notebooks/12_real_world_applications__emg_sign_classification_in_spd_manifold.ipynb

```diff
@@ -561,88 +561,6 @@
     "warnings.filterwarnings(\"ignore\")"
```

Comment:
<p>Ah super! You only removed the parts that use tf and keras: perfect! </p><p> </p><p>Could you adapt the numbering of the sections? I think A.1, A.2, A.3 got messed up. Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1778/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
See comment below about notebooks.